admin_manual/configuration/server/security/ransomware-protection.rst

```diff
@@ -74,6 +110,7 @@ Requirements
 #. **Ransomware Protection.** Ransomware protection needs to be in operation before an attack occurs, as it needs to record file operations to be able to revert them, in case of an attack.
 #. **ownCloud Versions App.** Required to restore older file versions. The capabilities of Ransomware Protection depend on its configuration regarding version retention.
 #. **ownCloud Trash Bin App.** Required to restore deleted files. The capabilities of Ransomware Protection depend on its configuration regarding trash bin retention.
+#. **Activity app.** This is only required for viewing activity logs.
```

Comment:
Forgot to say that before, sorry: Activity app is not an actual requirement for Ransomware Protection so I find it confusing at this place. It just displays e.g. restoration events in a user's activity timeline.

Reply:
Then it sounds fine as is. The message should indicate that it's optional. However, I'll revisit it just to be sure.